apps/login-register/src/app/register/register.component.ts

```diff
@@ -9,6 +11,30 @@ import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 export class RegisterComponent implements OnInit {
   registerForm: FormGroup = new FormGroup({});
 
+  errorMessages: any = {
```

Comment:
extract this into a constats.ts file to clear the cluttering

Reply:
I tried but doesn't work like expected, so I changed the logic there